app/views/registrants/show.html.haml

```diff
@@ -1,10 +1,12 @@
+= render :partial => 'layouts/weglot'
+
 - if !@registrant || @registrant.short_form?
   .intro-message
     %p=t('txt.registration.intro_message')
   
 %h1= t('txt.registration.step_1_header')
 
-#other_language_link
+#other_language_link{ style: 'display: none;' }
```

Comment:
is the mechanism for changing locales changing?

Reply:
We can't have both cause they conflict so hiding the existing locale changer within the ui.